public/app/plugins/datasource/grafana-azure-monitor-datasource/components/LogsQueryEditor/LogsQueryEditor.tsx

```diff
@@ -38,6 +40,14 @@ const LogsQueryEditor: React.FC<LogsQueryEditorProps> = ({
           variableOptionGroup={variableOptionGroup}
           onQueryChange={onChange}
           setError={setError}
+          selectableEntryTypes={[
+            ResourceRowType.Subscription,
+            ResourceRowType.ResourceGroup,
+            ResourceRowType.Resource,
+            ResourceRowType.Variable,
+          ]}
+          setResource={setResource}
+          resourceUri={query.azureLogAnalytics?.resource}
```

Comment:
do we need to pass along resourceURI if we pass along the query? Also I would thinking we don't need to send along setResource because I would guess that onChange does that but maybe I'm wrong?

Reply:
The `onChange` uses the return value of `setResource`. The `setResource` for metric queries is different for the log queries.

We might be able to not have to pass resource URI though. We might be able to use `query.azureLogAnalytics?.resource || query.azureMonitorMetrics?.resourceUri` in resource field. I'll take a look at this.